sprout.yaml

```diff
@@ -227,25 +227,6 @@ resources:
             etcd_cluster=$etcd_ip
             EOF
 
-            # Create /etc/chronos/chronos.conf.
-            mkdir -p /etc/chronos
-            cat > /etc/chronos/chronos.conf << EOF
-            [http]
-            bind-address = __private_sig_ip__
-            bind-port = 7253
-            threads = 50
-
-            [logging]
-            folder = /var/log/chronos
-            level = 2
-
-            [alarms]
-            enabled = true
-
-            [exceptions]
-            max_ttl = 600
-            EOF
-
             # Now install the software.
             DEBIAN_FRONTEND=noninteractive apt-get install sprout-node --yes --force-yes
```

Comment:
Richard mentioned that in the split-storage architecture, the sprout-node will be updated to contain only Sprout, so installing node package for all nodes will work fine.